lib/node/node.ts

```diff
@@ -422,6 +426,13 @@ export abstract class Node {
 	 */
 	abstract clone(): Node
 
+	/**
+	 * String representation of the node
+	 */
+	toString(): string {
```

Comment:
Should rather be `toJSON` so that it natively supports `JSON.stringify(node)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#description

Reply:
Done